src/structure.cpp

```diff
@@ -15,6 +15,7 @@
 %  Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 */
 
+#include <iostream>
```

Comment:
This should probably be removed (see comment below).

Reply:
Done